runners/wikiConsole/templates/npmInstall.asciidoc

```diff
@@ -0,0 +1,19 @@
+Install npm
+===========
+
+Prerequisites
+~~~~~~~~~~~~~~
+First make sure to have installed npm. We recommend to install the devonfwIDE which includes an npm installation. Alternatively, you can download npm https://www.npmjs.com/get-npm[here] and follow the installation instructions. 
+Then, you need to be located in the project directory where the package.json file lies.
```

Comment:
You can use the npmInstall command without project directory, too. For example if you want to install a package globally